app/src/main/java/com/ravenwallet/tools/sqlite/BRSQLiteHelper.java

```diff
@@ -44,8 +44,8 @@ public static BRSQLiteHelper getInstance(Context context) {
         return instance;
     }
 
-    public static final String DATABASE_NAME = "ravenwallet.db";
-    private static final int DATABASE_VERSION = 15;
```

Comment:
@ROSHiix Why the database version change (downgrade) and the database name change? This seems problematic. Have you tried an upgrade scenario with these changes?

Reply:
The database downgrade was done when we moved from ravencoin wallet classic to the one with assets.